mgcpy/independence_tests/abstract_class.py

```diff
@@ -10,6 +10,9 @@
 from scipy.stats import t
 
 
+def EUCLIDEAN_DISTANCE(x): return squareform(pdist(x, metric='euclidean'))
```

Comment:
this was done because parallel code need its parameter to be picklable, and unless a function is defined on the top level of the class it is not picklable